src/clarity-angular/nav/clrResponsiveNavigationService.ts

```diff
@@ -4,23 +4,30 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {Injectable} from "@angular/core";
-import {BehaviorSubject} from "rxjs/BehaviorSubject";
 
 import { ClrResponsiveNavCodes } from "./clrResponsiveNavCodes";
 import { ClrResponsiveNavControlMessage } from "./clrResponsiveNavControlMessage";
+import {Subject, Observable} from "rxjs";
```

Comment:
Sorry I completely missed this. I am relying too much on the Webstorm autocomplete. Will fix it. Thanks for catching this.